vnext/src/Libraries/Modules/Alert/Alert.uwp.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+'use strict';
+
+import {NativeModules} from 'react-native';
+const AlertNative = NativeModules.Alert;
```

Comment:
Please create a github issue to track moving this code into the central Alert.js and put a TODO comment in this file linking to it.